--- origsrc/clutter-gtk-1.0.0/clutter-gtk/Makefile.am
+++ src/clutter-gtk-1.0.0/clutter-gtk/Makefile.am
@@ -47,7 +47,8 @@ source_h_private = \
 libclutter_gtk_@CLUTTER_GTK_API_VERSION@_la_SOURCES = $(source_c) $(source_h) $(source_h_private)
 libclutter_gtk_@CLUTTER_GTK_API_VERSION@_la_LIBADD  = $(CLUTTER_LIBS) $(GTK_LIBS)
 libclutter_gtk_@CLUTTER_GTK_API_VERSION@_la_LDFLAGS = \
-	$(CLUTTER_LT_LDFLAGS) \
+	$(CLUTTER_GTK_LT_LDFLAGS) \
+	-no-undefined \
 	-export-symbols-regex "^gtk_clutter.*"
 
 cluttergtkheadersdir = $(includedir)/clutter-gtk-@CLUTTER_GTK_API_VERSION@/clutter-gtk
 
